packages/legend-art/style/base/_input.scss

```diff
@@ -55,7 +56,21 @@
   }
 }
 
+.input--caution {
+  border: 0.1rem solid var(--color-yellow-100);
+
+  &:hover {
+    border: 0.1rem solid var(--color-yellow-200);
+  }
+
+  &:focus {
+    border: 0.1rem solid var(--color-yellow-200) !important;
+  }
+}
+
 .input-group {
+  @include flexCenter;
+
```

Comment:
@xannem please be careful while making such changes. Because components in legend-art are used widely in other packages we have to make sure we are not breaking anything existing by making such styling changes. If you want this css at particular editor  I would suggest  you to customize it at editor level.